src/sky.h

```diff
@@ -133,12 +133,16 @@ class Sky : public scene::ISceneNode
 	bool m_clouds_visible; // Whether clouds are disabled due to player underground
 	bool m_clouds_enabled = true; // Initialised to true, reset only by set_sky API
 	bool m_directional_colored_fog;
+	bool m_use_old_skybox; // Enable to display the classic MT skybox
+	bool m_use_old_colors; // Enable to use pre 5.0 sky colours
 	bool m_bodies_visible = true; // sun, moon, stars
-	video::SColorf m_bgcolor_bright_f = video::SColorf(1.0f, 1.0f, 1.0f, 1.0f);
```

Comment:
Actually, since in MT master we have
`video::SColor cloudyfogcolor = m_bgcolor;`
It seems clearest to change `bgcolor` to `cloudyfogcolor` everywhere. I agree with you that `bgcolor` is unclear and confusing. But please keep `skycolor` unchanged as it is accurate and familiar. So then the names are improved but changes are minimised.